web/styles/settings.css

```diff
@@ -1846,7 +1854,7 @@ label.preferences-radio-choice-label {
     }
 }
 
-@media (width < 830px) and (width > $md_min) {
+@container (width < 830px) and (width > $settings_overlay_sidebar_collapse_breakpoint) {
     #attachments-settings .upload-file-name {
         min-width: 10em;
```

Comment:
Just studying the code again, but I wonder why we need such a complex query for setting a `min-width`? Having one condition here in px and the other in ems makes this really hard to think about.

Reply:
Looking at attachment settings as an example, it seems like the max cut-off isn't really necessary, we could just do a min width for this. It's just guaranteeing that columns get enough space, and with wider screens then there's even more space.

Reply:
Added a commit for that at the start of this PR